packages/react/src/Accordion/Accordion.tsx

```diff
@@ -28,7 +28,7 @@ const AccordionRoot = forwardRef(
     const { keyDown } = useKeyboardFocus(innerRef, { rotating: true })
 
     return (
-      <AccordionContext.Provider value={{ headingLevel: headingLevel, section: section }}>
+      <AccordionContext.Provider value={{ headingLevel: headingLevel, as: sectionAs }}>
```

Comment:
I suggest not renaming at this hierarchical level and only using `as` within `Section`.